db/controllers/reviews.controller.js

```diff
@@ -2,11 +2,15 @@ const { fetchReview } = require("../models/reviews.model");
 
 exports.getReview = (request, response, next) => {
   const reviewToFetch = request.params;
-  fetchReview(reviewToFetch)
-    .then((fetchedReview) => {
-      response.status(200).send(fetchedReview);
-    })
-    .catch((err) => {
-      next(err);
-    });
+  if (reviewToFetch.review_id.match(/([0-9])+/)) {
+    fetchReview(reviewToFetch)
+      .then((fetchedReview) => {
+        response.status(200).send(fetchedReview);
+      })
+      .catch((err) => {
```

Comment:
You've passed the next parameter to getReview, if we get to the catch block there is no doubt that there is an error. There is no need for a callback function checking if an error exists, by simply getting to the catch block there is an error. Simply passing next here is enough.

Reply:
Super helpful thank you!